dll/cpl/intl/lang/ro-RO.rc

```diff
@@ -125,21 +125,21 @@ CAPTION "Oră"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Exemplu", -1, 7, 7, 230, 33, WS_CHILD | WS_VISIBLE
-    LTEXT "Oră:", -1, 13, 21, 54, 10, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Exemplu de notare oră:", -1, 13, 21, 54, 10, WS_CHILD | WS_VISIBLE | WS_GROUP
     EDITTEXT IDC_TIMESAMPLE, 68, 19, 84, 14, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
-    LTEXT "For&matul de reprezentare a orei:", -1, 13, 52, 130, 10, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "&Format oră:", -1, 13, 52, 130, 10, WS_CHILD | WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEFORMAT, 149, 50, 84, 100, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Sepa&ratorul din reprezentarea orei:", -1, 13, 70, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "&Separator pentru oră:", -1, 13, 70, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMESEPARATOR, 149, 68, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Simbolul A.&M.:", -1, 13, 88, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "Simbol A&M:", -1, 13, 88, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEAMSYMBOL, 149, 86, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Sim&bolul P.M.:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "Simbol &PM:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEPMSYMBOL, 149, 104, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "", -1, 7, 140, 230, 80, WS_VISIBLE
-    LTEXT "Notația orei:\n\n\
-h = oră     m = minut    s = secundă\nt = am sau pm\n\
-h = 12 ore    H = 24 ore\n\
-hh, mm, ss = cu zerouri   h, m, s = fără zerouri", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Notarea formatelor de oră\n\n\
```

Comment:
Keep this layout in sync with en-US, if it doesn't fit visually, tweak the coordinates.

Reply:
I see now. Thank you @binarymaster!